examples/capture/index.html

```diff
@@ -345,17 +348,31 @@ <h3 style="text-align:left;margin-left:15px;">Macros</h3>
     width: 640
   }) 
 
-
   setInterval($W.getRow,100)
   	$W.calibrated = false
 
   if ($('body').width() > 768) $('#tool-toggle').hide()
   else $('#tool-pane').hide()
-
-  })(); 
+  })();
+
+//code to connect to raspberry-pi
+  var el=document.getElementById("pi-button");
+  if(el){
+    //if pi_div button does not return null
+document.getElementById("pi-button").addEventListener("click", function connect_to_pi() {
+    $W.loadingfromPi=true;
+    var piImage = new Image();
```

Comment:
As to what I wrote above about `$W.piImage`, here's a suggestion:

```suggestion
    $W.piImage = new Image();
```

And we have to change all instances of `piImage` to `$W.piImage` below too.

Reply:
Have implemented this.